app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -94,45 +94,45 @@ const TopicTrackingState = EmberObject.extend({
     this.messageBus.subscribe(
       "/latest",
       this._processChannelPayload,
-      meta["/latest"] || messageBusDefaultNewMessageId
+      meta["/latest"] ?? messageBusDefaultNewMessageId
```

Comment:
@OsamaSayegh I found the bug. We were sending down `0` as the last message id for the channel but because `0 || -1` returns `-1`, we ended up telling this subscription not to catch up with the backlog.

Reply:
Ohh nice catch, thank you for looking into it! 🙇